tests_operations.py

```diff
@@ -61,4 +61,5 @@ def test_modulo():
     WHEN:  passed in to the function modulo
```

Comment:
Please add the function definition otherwise the build will fail if we merge it in